packages/contracts/src/ubiquistick/UbiquiStickSale.sol

```diff
@@ -101,18 +101,20 @@ contract UbiquiStickSale is Ownable, ReentrancyGuard {
 
         if (remainingTokenCount < count) {
             count = remainingTokenCount;
+            paid = remainingTokenCount;
         }
         if (msg.value < count * price) {
+            paid = (msg.value * price) / price;
```

Comment:
What is the purpose of multiplying and dividing to the same `price` variable ?

Reply:
> What is the purpose of multiplying and dividing to the same `price` variable ?

<img width="284" alt="image" src="https://github.com/ubiquity/ubiquity-dollar/assets/125548724/121bd452-509d-499c-9e9d-adc5fdd6b4fe">
the count calculated using the msg.value/ price and then there was paid variable in the bottom which was multiplying the count with price but the slither wants us to multiply the price first and then divide the thing but putting paid = count/ price and gonna break stuff so just to make sure that paid gets the right value i thought to change the paid variable in the conditional functions

Reply:
Can't we refactor to:
```
paid = msg.value;
```

?

Reply:
> Can't we refactor to:
> 
> ```
> 
> paid = msg.value;
> 
> ```
> 
> 
> 
> ?

Sure my bad I didn't look 😅😅😅we can